modules/configure-slack.adoc

```diff
@@ -15,9 +15,9 @@ Start by creating a new Slack app, and get the webhook URL.
 +
 [NOTE]
 ====
-If you want to use an existing Slack app, go to https://api.slack.com/apps and select an app.
+If you want to use an existing Slack app, go to link:https://api.slack.com/apps and select an app.
```

Comment:
This and the link on line 20 don't have brackets afterward with text, so it's not rendering properly as a link (and is actually showing "link:" in the docs